include/nbl/ext/FullScreenTriangle/FullScreenTriangle.h

```diff
@@ -44,12 +57,17 @@ inline auto createFullScreenTriangle(asset::IAssetManager* am, video::IVideoDriv
     return retval;
 }
 
-inline auto createFullScreenTriangle(core::smart_refctd_ptr<video::IGPUSpecializedShader>&& fragShader, core::smart_refctd_ptr<video::IGPUPipelineLayout>&& pipelineLayout, asset::IAssetManager* am, video::IVideoDriver* driver, const asset::SBlendParams& blendParams={}, const asset::SRasterizationParams& rasterParams={})
+inline auto createFullScreenTriangle(nbl::video::IGPUObjectFromAssetConverter::SParams& cpu2gpuParams, core::smart_refctd_ptr<video::IGPUSpecializedShader>&& fragShader, core::smart_refctd_ptr<video::IGPUPipelineLayout>&& pipelineLayout, const asset::SBlendParams& blendParams={}, const asset::SRasterizationParams& rasterParams={})
```

Comment:
this is all messed up, there shouldn't really be a triangle mesh anymore (cause it has no vertices, nothing), just:
- create a proto pipeline for it in one function
- create a full pipeline given proto, frag shader and pipeline layout
- provide a helper function for drawing the tri (without a meshbuffer)

Reply:
corrected https://github.com/Devsh-Graphics-Programming/Nabla/pull/172/commits/555cc6654cc2ed7b8b1c94d081361753696ef389